packages/components/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -14,6 +14,10 @@ export interface ProgressBarProps {
   backgroundColor?: SystemProp<keyof Theme["colors"], Theme>;
   /** Sets the size of the progress bar. */
   size?: ProgressBarSizeOptions;
+  /** The style of the progress indicator background color, should be an object with html style attributes */
+  style?: React.CSSProperties;
```

Comment:
This prop is not being used in the component, right?

Reply:
By having this:
```
<ProgressBar
      backgroundColor="colorAvatarBackgroundPink"
      indicatorStyles={{
        backgroundColor: "purple",
      }}
      value={25}
    />
```
I was able to see this: 

![image](https://user-images.githubusercontent.com/5320963/227988993-d54f21be-dc72-43d1-83b5-a459924e1f7a.png)

The backgroundColor of the indicator is the style that you are already adding as prop.

Reply:
Actually there is a backgroundColor for the progress already made and backgroundColor for the progress not done yet.

Reply:
I don't know what is the deal here but TS is complaining 😞 

![Screen Shot 2023-03-27 at 12 31 38](https://user-images.githubusercontent.com/2047941/227989133-9329269d-78a8-4287-8de6-52e4b76c0195.png)

Reply:
Yes but Ana now try overwriting the pink background color

Reply:
Why are we adding an `style` property?

Reply:
to be possible to overwrite styles from a token with any custom ones

Reply:
It would solve the problem, right? https://github.com/Localitos/pluto/pull/682#discussion_r1149447921

Reply:
But it will create a mess on the talent app. Because we will need to replace colors in all components. Besides that it will also create a mess on the DS. All components will be too open to adding any styles, we can easily lose track of what we want for the DS, and if a token changes, we will need to figure out every place where styles were added. 

If you use tokes to defined it You just need to map on the talent app:

```typescript
{
  tokenX : colorPrimaryFromBackend
}
```

Reply:
This means that the user can add the color, they don't need to add the token. The BR will store the colors and the FE will map wich color should be added to which token.

Reply:
We can add the tokens you need on Pluto instead of adding `styles` tags in all components we need to change. But my suggestion is not to create tokens for specific components but to map what colors you need to change. For example:

You need to update the backgroundColor from `<ProgressBar />`  and `<Button />` and currently they use the token `colorBackground`. 

But the `<Box.div backgroundColor="colorBackground">` uses the same token. And you don't want to change it. 

My suggestion: create a new token, for example `colorBackgroundPrimaryComponents` and use it in all components you want to apply the theming.

Of course,  `colorBackground` and `colorBackgroundPrimaryComponents`  would have the same token on Pluto, but you would be able to override only the one you need.